File: TRClient/src/app/Entities/LoadMonthlyWrapper.ts
import {LoadMonthlyItem} from './LoadMonthlyItem';
import {Awrapper}        from './Awrapper';

export class LoadMonthlyWrapper extends Awrapper {
   
    AvailLogEntries: LoadMonthlyItem[] = [];
    FileImported: number = -1;
  
 }

File: TRBusinessLayer/DataObjects/LoadMonthlyWrapper.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace TRBusinessLayer.DataObjects
{
    public class LoadMonthlyWrapper : AbstractDto
    {
      public bool FileImported { get; set; }
      public  List<LogEntry> AvailLogEntries { get;set; }
    }
}